src/containers/SaveBarHandler/SaveBarButton/index.jsx

```diff
@@ -1,19 +1,24 @@
 import styles from '../style.postcss';
 
 import React from 'react';
+import classnames from 'classnames';
 import Button from 'components/Button';
 import PropTypes from 'prop-types';
+import testClass from 'domain/testClass';
 
-const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) =>
-  <Button autoWidth
+const SaveBarButton = ({ label, isPrimary, disabled, onClick, linkHref, linkTo }) => {
+  const className = classnames(styles.SaveBar_button,
+    testClass(`${label.replace(/\s+/g, '')}-btn`));
```

Comment:
Another thing is that we want these classes to be as unique as possible. How about `test-toolbar-button-<label>` instead?